sql/core/src/main/scala/org/apache/spark/sql/execution/ui/SQLTab.scala

```diff
@@ -18,15 +18,22 @@
 package org.apache.spark.sql.execution.ui
 
 import org.apache.spark.internal.Logging
+import org.apache.spark.sql.execution.streaming.StreamQueryStore
+import org.apache.spark.sql.streaming.ui.{StreamingQueryPage, StreamingQueryStatisticsPage}
 import org.apache.spark.ui.{SparkUI, SparkUITab}
 
-class SQLTab(val sqlStore: SQLAppStatusStore, sparkUI: SparkUI)
+class SQLTab(
```

Comment:
As your comment https://github.com/apache/spark/pull/26201#issuecomment-544823816, I also think it is imperative to create a new tab for this, "Structure Streaming" or "Streaming Query" maybe a good name.